### PR TITLE
Update & clear environment variable

### LINKED ISSUE #24: Add an interface for updating env vars across a bunch of apps
Create an interface something like this:

    # set an env
    ec app1 app2 --key=MY_KEY --value=bar
    # clear an env
    ec app1 app2 --key=MY_KEY

### PR DESCRIPTION
**Pull request by jaydorsey**
Allows adding, updating, deleting environment variables across multiple
Heroku apps.

Closes #24

### CODE REVIEW

**Comment by smridge** [comment]
This is nice! I haven't tested locally, but I assume this works.

File: lib/env_compare/cli.rb [hunk 1]
```diff
@@ -103,5 +113,9 @@ def theme(file)
     def theme_path(file)
       @theme_path ||= File.join(File.dirname(__dir__), '..', 'themes', "#{file}.erb")
     end
+
+    def update_config(app, body)
```
Comment (smridge) [inline lib/env_compare/cli.rb:117]: Maybe changing `body` to `hash`?

Comment (jaydorsey) [reply lib/env_compare/cli.rb:117]: Good idea; let me do that as a follow. I think I'm going to split these out into different files shortly

I'm going to use this in about 5 minutes 😂

File: README.md [hunk 2]
```diff
@@ -63,6 +63,8 @@ Use `ec` command to start comparing 2 or more heroku application environment var
 
 When only one application name is specified, output is shown in cli. Otherwise, your default browser is launched.
 
+## Compariing environment variables
```
Comment (smridge) [inline README.md:66]: Typo 🙃

**Pull request merged**